### PR TITLE
doc(contributing.md): add rpm section, update deb cmds

### PR DESCRIPTION
**Pull request by hahnandrew**
*Description of changes:*

contributing doc maintenance, updating glibdev install section

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Jordan-Nelson — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,9 +110,13 @@ Some platforms require additional setup. See below for details.
 `libsecret` and `glib` are required for some plugins. They can be installed on Debian based linux distributions such as Ubuntu by running the command below.
 
 ```sh
-sudo apt-get update && sudo apt-get install libsecret-1-dev libglib2.0-dev
+sudo apt update && sudo apt install libsecret-1-dev libglib2.0-dev
```
Comment (reesscot) [inline CONTRIBUTING.md:113]: Did this command change in the latest versions of Ubuntu?

Comment (hahnandrew) [reply CONTRIBUTING.md:113]: No, they're different frontends for the same package manager - `apt` is facto _(note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))_

Comment (hahnandrew) [reply CONTRIBUTING.md:113]: No, they're different frontends for the same package manager - apt is de facto (note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))

Comment (hahnandrew) [reply CONTRIBUTING.md:113]: No, they're different frontends for the same package manager - `apt` is de facto (note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))

Comment (hahnandrew) [reply CONTRIBUTING.md:113]: aptNo, they're different frontends for the same package manager - `apt` is de facto _(note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience)_)

Comment (hahnandrew) [reply CONTRIBUTING.md:113]: No, they're different frontends for the same package manager - `apt` is de facto _(note: !=alias, read more [here](https://aws.amazon.com/compare/the-difference-between-apt-and-apt-get/#:~:text=apt%2Dget%20update-,Key%20differences%3A%20apt%20vs.,management%20tools%20for%20user%20convenience))_

Comment (reesscot) [reply CONTRIBUTING.md:113]: thanks for the link, lgtm

**Review by Jordan-Nelson — APPROVED** [review]

**Review by tyllark — APPROVED** [review]

**Pull request merged**